packages/flutter_driver/test/src/real_tests/extension_test.dart

```diff
@@ -19,7 +19,11 @@ import 'package:flutter_driver/src/common/text.dart';
 import 'package:flutter_driver/src/common/wait.dart';
 import 'package:flutter_driver/src/extension/extension.dart';
 import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/mockito.dart';
```

Comment:
nit: mockito based tests are going to become more difficult to manage as we move to null-safety. If there's any way to avoid using mockito here and instead either using `Fake` from `package:test` or just implementing our own fake version, it will make future migrations for this file easier.